_posts/2021-05-04-lessons.md

```diff
@@ -0,0 +1,20 @@
+
+My approach as a music teacher is not "cookie-cutter". I have a few concepts that <!--more--> apply to every student; learning to read music is one of the big ones, as well as learning how to “practice smart”, and, being the percussionist that I am, learning to have good rhythm is of course very important! But other than these few things, lessons differ based on my students’ interests and goals. This includes a range of using method books, popular songs and tunes they like, improvising, learning chords, theory and even sometimes composing. It’s all about having a good balance of challenge but also fun. I include basic theory in our lessons so the students learn the “why” behind learning scales and chords, and the fundamentals and foundations of music. All things which I found lacking in my own experience learning music. I started taking music lessons when I was 4, and my memories include lots of crying from mean, harsh teachers (one of whom my family hilariously dubbed "Dr. No-No"). I think learning music should be fun, something you enjoy doing and can be proud of. If you want more of a challenge, I am here to give it to you. But it is up to you, the student, how far you go based on how much time you are putting into practicing and developing your music-making outside of our one-on-one lessons. I want my students to feel like they CAN play their instrument well, and feel proud that they have learned the tools they need to be a self-starter at home, learning music on their own time. And I'm here to help coach them along the way. Music became a career for me, but for my students it is most important that they have a great experience so that music can be something they carry with them for the rest of their lives, whether it is a lifelong hobby or a career.
```

Comment:
My approach as a music teacher is not "cookie-cutter". I have a few concepts that

`<!--more-->`

apply to every student; learni....

Reply:
learning to read music is one of the big ones, as well as learning how to “practice smart”. And as a percussionist teaching how to have good rhythm is of course very important!

Reply:
theory in our lessons so _my_ students learn the “why” behind learning scales

Reply:
and my memories include lots of hard times through challenging teachers (one of whom my family hilariously dubbed "Dr. No-No")